documentation/modules/rn-2.2.adoc

```diff
@@ -95,4 +95,6 @@ The error status message for a VM with no operating system on the *Migration pla
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
 
-.If a VM resource is deleted during the image conversion stage, the VM's migration is still marked as successful.
\ No newline at end of file
+.If a target VM resource is deleted during the image conversion stage, the migration of the VM is still marked as successful.
+
+If you delete a target VM CR during the 'Convert image to kubevirt' stage of the migration, the MTV displays an error, `VirtualMachine CR not found`. However, the status of the VM migration appears as `Succeeded` in the `plan.yaml` file and in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2031529[*BZ#2031529*])
```

Comment:
What is the name of the VM CR? If it is "VirtualMachine" CR, that should be used instead of "VM" because that is the proper name of the CR.

"the MTV displays an error"? What is displaying the error?